src/com/dre/brewery/Brew.java

```diff
@@ -332,12 +336,21 @@ public void distillSlot(ItemStack slotItem, PotionMeta potionMeta) {
 
 			addOrReplaceEffects(potionMeta, getEffects(), quality);
 			potionMeta.setDisplayName(P.p.color("&f" + recipe.getName(quality)));
-			slotItem.setDurability(PotionColor.valueOf(recipe.getColor()).getColorId(canDistill()));
+			if (P.use1_9) {
+				potionMeta.setMainEffect(PotionColor.valueOf(recipe.getColor()).getEffect());
+			} else {
+				slotItem.setDurability(PotionColor.valueOf(recipe.getColor()).getColorId(canDistill()));
+			}
+
 		} else {
 			quality = 0;
 			removeEffects(potionMeta);
 			potionMeta.setDisplayName(P.p.color("&f" + P.p.languageReader.get("Brew_DistillUndefined")));
-			slotItem.setDurability(PotionColor.GREY.getColorId(canDistill()));
+			if (P.use1_9) {
+				potionMeta.setMainEffect(PotionColor.GREY.getEffect());
+			} else {
+				slotItem.setDurability(PotionColor.GREY.getColorId(canDistill()));
+			}
```

Comment:
The Distill Tracker does take care of that -- see here https://github.com/DieReicheErethons/Brewery/pull/116/files#diff-53aa04721776ae80799c7b0e613da93bR165

This is called before any attempt to distill a brew is made, and is checked during the brewing process to make sure no-one has taken all the valid brews out of the brewing stand.